deltadb/src/python-old/README

```diff
@@ -7,7 +7,7 @@ Overview
 
  When servers report their status to the catalog server, a log file is created to track those reports. The servers are of different types and are allowed to report any values for any field name. In other words, there is no schema beyond the requirement that each reported item must contain a field and value. The catalog server also creates a daily checkpoint file with the current status of all servers still actively reporting. Samples of data from checkpoints and log files for different types of servers are shown further down in this document. The following scripts operate on this data for the creation of summary reports.
 deltadb_collect: Returns log data for the specified time period. The script takes three arguments. The first argument is a directory where the catalog history files are stored in sub-directories by year. The second argument specifies the start time as a timestamp, date or time in the format YYYY-MM-DD-HH-MM-SS. The third argument (optional) can be a timestamp, duration, or date and time of the same format YYYY-MM-DD-HH-MM-SS. The duration can be in seconds (+180 or s180), minutes (m45), hours (h8), days (d16), weeks (w3), or years (y2).
-deltadb_select_*: Filters streamed data (through standard input) and returns only the objects which satisfy the provided arguments. For example, type=wq_master would only output objects with type wq_master. There are two select tools: deltadb_select_static, deltadb_select_dynamic. The man pages describe the differences between these tools in more detail.
```

Comment:
I don't think we want to change the values in this deltadb example, because `wq_master` is the historical value in the database.  The `type` field in the catalog report hasn't changed, right?